### DISCUSSION TITLE
Pymobiledevice3 windows executable file

### DISCUSSION THREAD

**Discussion by mega6453** [doronz88/pymobiledevice3#1033] [General]
I want pymobiledevice3 as a single windows executable which includes the python runtime also inside it. So, no need to install the python in all the machines.  Basically I want to use the single pymobiledevice3 in the software which I'm developing for mobile automation. So, single exe file will be helpful for me to use the commands easily in the project. 

I've tried **auto-py-to-exe** and **pyinstaller** but I'm facing some issues. So any help please?
Thanks

**Comment by doronz88** [comment]
Please submit this as an issue - specifically the pyinstaller since this is documented and should work

**Reply by mega6453** [reply]
@doronz88 ok sure.
 I've a suggestion. Why don't you create a github action that creates the windows executable file whenever there's a new version released?
That will be helpful for people like me.. 
Thanks!

**Reply by doronz88** [reply]
I'm not a pyinstaller user myself (never used it)
If anyone is up for it, he's welcome to submit this as a PR
